flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -174,6 +173,20 @@
 				" help relieve back-pressure caused by unbalanced data distribution among the subpartitions. This value should be" +
 				" increased in case of higher round trip times between nodes and/or larger number of machines in the cluster.");
 
+	/**
+	 * Number of max buffers can be used for each output subparition.
+	 */
+	@Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+	public static final ConfigOption<Integer> NETWORK_MAX_BUFFERS_PER_CHANNEL =
+			key("taskmanager.network.max-buffers-per-channel")
```

Comment:
@Jiayi-Liao shouldn't this be `taskmanager.network.memory.max-buffers-per-channel` ?

Reply:
@pnowojski You're right! It's fixed in my latest commit.